cleanlab/multilabel_classification/filter.py

```diff
@@ -0,0 +1,228 @@
+import warnings
+from typing import Optional, Union, Tuple, List, Any
+import numpy as np
+
+
+def find_label_issues(
+    labels: list,
+    pred_probs: np.ndarray,
+    return_indices_ranked_by: Optional[str] = None,
+    rank_by_kwargs={},
+    filter_by: str = "prune_by_noise_rate",
+    frac_noise: float = 1.0,
+    num_to_remove_per_class: Optional[int] = None,
+    min_examples_per_class=1,
+    confident_joint: Optional[np.ndarray] = None,
+    n_jobs: Optional[int] = None,
+    verbose: bool = False,
+) -> np.ndarray:
+    """
+    Finds label issues in multi-label classification data where each example can belong to more than one class.
```

Comment:
```suggestion
    Finds label issues in multi-label classification data where each example can belong to more than one class.
    An example is flagged as with a label issue if *any* of the classes appear to be incorrectly annotated for this example.
```

Reply:
Minor doc changes in first line, added here: https://github.com/cleanlab/cleanlab/blob/25672dc504471dd2d450642942555e44528565dc/cleanlab/multilabel_classification/filter.py#L21